__tests__/app.test.js

```diff
@@ -65,10 +65,86 @@ describe("GET /api/articles/:article_id", () => {
   test("requests for invalid article IDs are rejected", () => {
     return request(app)
       .get("/api/articles/999999")
-      .expect(404)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.message).toBe("999999 is not a valid article ID.");
+      });
+  });
+});
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("article votes can be increased", () => {
+    const addedVotes = { inc_votes: 55 };
+    return request(app)
+      .patch("/api/articles/9")
+      .send(addedVotes)
+      .expect(200)
+      .then(({ body }) => {
+        expect(Object.keys(body).length).toBe(1);
+        expect.objectContaining({
```

Comment:
expect _what_ is an `objectContaining`?

you're missing some syntax here

make sure you're using the `expect` function and some sort of _matcher_ (i.e: `toEqual` etc.)